src/daft-proto/src/generated/daft.v1.rs

```diff
@@ -740,13 +740,14 @@ pub struct RelProject {
     #[prost(message, repeated, tag = "2")]
     pub projections: ::prost::alloc::vec::Vec<Expr>,
 }
-/// This does not need to be separate, but it reduces impedance mismatch across the type definitions.
 #[derive(Clone, PartialEq, ::prost::Message)]
-pub struct RelActorPoolProject {
+pub struct RelUdfProject {
```

Comment:
Talked to Conner and he said since this is in dev, its OK to just modify the proto directly